wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -123,22 +124,33 @@ def is_alive(self) -> bool:
             return False
         return True
 
-    def _try_load_job(self) -> bool:
-        _public_api = public.Api()
-        try:
-            _job_artifact = _public_api.artifact(self._sweep_config["job"], type="job")
-            wandb.termlog(
-                f"{LOG_PREFIX}Successfully loaded job: {_job_artifact.name} in scheduler"
+    def _try_load_executable(self) -> bool:
+        if self._sweep_config.get("job"):
+            _public_api = public.Api()
+            try:
+                _job_artifact = _public_api.artifact(
+                    self._sweep_config["job"], type="job"
+                )
+                wandb.termlog(
+                    f"{LOG_PREFIX}Successfully loaded job: {_job_artifact.name} in scheduler"
+                )
+            except Exception as e:
+                wandb.termerror(f"{LOG_PREFIX}{str(e)}")
+                return False
+            return True
+        elif self._sweep_config.get("docker_image_uri"):
+            # TODO(gst): check docker existance? Maybe confirm NOT wandb run uri?
```

Comment:
not sure how to check if a docker image actually exists, especially if the scheduler is running in local-process and launching jobs into a docker queue

Reply:
We'll need to leverage the registry config here. We have a utility function `docker_image_exists` (again, bad name), where we inspect the image locally. This could be implemented through our new Registry class being introduced in this PR: https://github.com/wandb/wandb/pull/4916

Would likely need to add a `LocalRegistry` or something, cc: @bcsherma

Reply:
Maybe in followup PR?

Reply:
Yep, follow up